security/src/main/java/org/aggregatortech/dindora/exceptions/ErrorMessageService.java

```diff
@@ -0,0 +1,42 @@
+package org.aggregatortech.dindora.exceptions;
+
+import java.util.HashMap;
+
+public class ErrorMessageService {
```

Comment:
We can make it a HK2 service by using the @Service annotation and inject it instead of using the getInstance() method

Reply:
yes I believe we should do it.. Are you okay with overall pattern?

Reply:
yes the overall pattern looks good, will merge it.